### DISCUSSION TITLE
Deployment Docker .env Not Work

### DISCUSSION THREAD

**Discussion by Kaseres** [lowdefy/lowdefy#1132] [Questions and Help] [answered]
Hi, 
I have problem for Deployment Docker .
I follow Link : https://docs.lowdefy.com/docker
But when docker run -p 3000:3000 <tag> 
Result file .env Not work

How i build Dockerfile give .env working for Docker

**Comment by SamTolmay** [comment, marked as answer]
Hi @Kaseres 

Please see this discussion: https://github.com/lowdefy/lowdefy/discussions/970
